roles/ocm-labels/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+ocm_merge_type: preserve
+ocm_protect_keys: true
+ocm_protected_keys:
+  - 'name'
+  - 'clusterID'
+  - 'cloud'
+  - 'vendor'
+  - 'local-cluster'
+  - 'openshiftVersion'
+  - 'installer.name'
+  - 'installer.namespace'
+  - 'feature.open-cluster-management.io/addon-application-manager'
+  - 'feature.open-cluster-management.io/addon-cert-policy-controller'
+  - 'feature.open-cluster-management.io/addon-iam-policy-controller'
+  - 'feature.open-cluster-management.io/addon-policy-controller'
+  - 'feature.open-cluster-management.io/addon-search-collector'
+  - 'feature.open-cluster-management.io/addon-work-manager'
```

Comment:
is it possible to do wild card here?

Reply:
Yes, wildcards are possible. I understand wildcards are future-proofing, but I opted for explicit definitions for clarity.

Reply:
Wildcards/regex support added.